prime-router/src/test/kotlin/fhirengine/translation/hl7/FhirTransformerTests.kt

```diff
@@ -605,4 +607,37 @@ class FhirTransformerTests {
         assertThat(servRequest1.id).isEqualTo("0")
         assertThat(servRequest2.id).isEqualTo("1")
     }
+
+    @Test
+    fun `test transform dateTime property with changeTimezone`() {
+        val bundle = Bundle()
+        bundle.id = "abc123"
+        val isoFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssXXX")
+        bundle.timestamp = isoFormat.parse("2021-08-09T08:52:00-04:00")
+        val resource = Patient()
+        resource.id = "def456"
+        resource.meta = Meta()
+        bundle.addEntry().resource = resource
+
+        val elemA = FhirTransformSchemaElement(
+            "elementA",
+            constants = sortedMapOf(Pair("timezone", "'America/Phoenix'")),
+            value = listOf("Bundle.timestamp.changeTimezone(%timezone)"),
+            resource = "Bundle.entry.resource.ofType(Patient).meta",
+            bundleProperty = "Bundle.entry.resource.ofType(Patient).meta.lastUpdated"
+        )
+
+        val schema = FhirTransformSchema(elements = mutableListOf(elemA))
+
+        FhirTransformer(schema).transform(bundle)
+        val newValue =
+            FhirPathUtils.evaluate(
+                CustomContext(bundle, bundle),
+                bundle,
+                bundle,
+                "Bundle.entry.resource.ofType(Patient).meta.lastUpdated"
+            )
+        assertThat(resource.meta.lastUpdated).isEqualTo(bundle.timestamp)
+        assertThat(newValue[0].primitiveValue()).isEqualTo("2021-08-09T05:52:00.000-07:00")
```

Comment:
Add tests with partial date times:
1. What happens if there is only a year?
2. What happens if there is only a date (1/1/2023)?
Then figure out if that behavior is correct.